src/app/components/machinechart/machinechart.component.ts

```diff
@@ -41,15 +42,14 @@ export class MachinechartComponent implements OnInit {
 
 		setTimeout(() => {
 			try {
-				// @ts-ignore
-				if (this.chartData && this.chartData.length == 1 && this.chartData[0] == 'system_missing') {
+				if (this.chartData && this.chartData.Data.length == 1 && this.chartData.Error == 'system_missing') {
 					this.specificError = 'system_missing'
 					this.chartError = true
 					return
 				}
 				this.doChart(this.key, this.id, this.chartData)
-				//@ts-ignore
-				this.chartError = this.chartData.length <= 0 || this.chartData[0].data.length <= 1
+
+				this.chartError = this.chartData.Data.length <= 0 || this.chartData.Data[0].data.length <= 1
 			} catch (e) {
 				this.chartError = true
```

Comment:
This should contain a log entry, should it not?

Reply:
fixed